src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20240916141644_EES4490_AddThemeToPublication.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    /// <inheritdoc />
+    public partial class EES4490_AddThemeToPublication : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Guid>(
+                name: "ThemeId",
+                table: "Publications",
+                type: "uniqueidentifier",
+                nullable: false,
+                defaultValue: Guid.Empty);
+
+            migrationBuilder.CreateIndex(
+                name: "IX_Publications_ThemeId",
+                table: "Publications",
+                column: "ThemeId");
+
+            migrationBuilder.DropForeignKey(
```

Comment:
Is it necessary to drop the foreign key at this point, or is it worth deferring this until we actually come to remove this column?

Reply:
Probably not, I'll remove it